packages/docs/components/atoms/CircularMarquee/stories/app.twig

```diff
@@ -0,0 +1,8 @@
+<div style="height: 400vh">
```

Comment:
**suggestion :** Add some space before and after the container to enable the scroll and hide the overflow to disable horizontal scroll

```twig
<div class="flex items-center justify-center h-48">
  Scroll down
</div>
<div class="h-screen overflow-hidden">
  {% include '@ui/atoms/CircularMarquee/CircularMarquee.twig' with {
    id: 'douceur',
    radius: 120,
    outer_radius: 150,
    content: ' Douceur • Fraîcheur • Douceur • Fraîcheur • Douceur • Fraîcheur • '
  } only %}
</div>
<div class="flex items-center justify-center h-screen">
  Scroll up
</div>
```